### PR TITLE
[RORDEV-975][RORDEV-238] the `indices` rule should allow kibana-related indices 

### PR DESCRIPTION
**Pull request by coutoPL**
when the `kibana` rule and the `indices` rule are defined in the same block, the `indices` rule should allow all Kibana-related indices implicitly, without additional configuration.

### CODE REVIEW

File: elasticsearch.md [hunk 1]
```diff
@@ -982,6 +982,18 @@ If used in conjunction with ReadonlyREST Enterprise, this rule enables **multi t
...
+**⚠️IMPORTANT** When you use the `kibana` rule together with the `indices` rule in the same block, you don't have to explicitly allow
+the Kibana-related indices in the list of allowed indices of the `indices` rule. ROR will do it for you automatically. 
+
+Example:
+```yaml
+- name: "::RW_USER::"
+  auth_key: rw_user:pwd
+  kibana:
+    access: rw
+  indices: ["r*"] # .kibana, .kibana_8.10.4, .kibana_task_manager, etc are allowed here, because there is the `kibana` rule in the same
```
Comment (Dzuming) [inline elasticsearch.md:994]: Isn't it something missing after `rule in the same`?

**Review by Dzuming — APPROVED** [review]

File: kibana.md [hunk 2]
```diff
@@ -8,21 +8,21 @@ description: User manual for ReadonlyREST Enterprise/PRO/Free
...
+ReadonlyREST plugin for Kibana is not open source, and it's offered as part of the [ReadonlyREST PRO](https://readonlyrest.com/pro.html) and [ReadonlyREST ENTERPRISE](https://readonlyrest.com/enterprise.html), and [ReadonlyREST Free](https://readonlyrest.com/free) packages. See product descriptions and a comparison chart on the official [ReadonlyREST website](https://readonlyrest.com)
 
-ReadonlyREST plugins for Kibana **always require** ReadonlyREST opens source plugin to be installed in the Elasticsearch nodes your Kibana instance(s) will connect to.
+ReadonlyREST plugins for Kibana **always require** the ReadonlyREST open-source plugin to be installed in the Elasticsearch nodes your Kibana instance(s) will connect to.
 
-It's not mandatory to install ReadonlyREST in all Elasticsearch nodes, but only in the ones in where you need the HTTP interface to be secured.
+It's not mandatory to install ReadonlyREST in all Elasticsearch nodes, but only in the ones, where you need the HTTP interface to be secured.
```
Comment (sscarduzio) [inline kibana.md:15]: the "in" was not supposed to be there, but I'm not sure why a comma was added?

File: kibana.md [hunk 3]
```diff
@@ -451,11 +451,11 @@ Imagine you run Elasticsearch and Kibana on the same host:
 * Credentials are wrong, and the first block does not match
 * The second block is then evaluated, and the request is allowed because of its origin IP
 
-As you can see, Elasticsearch has no user related information (metadata) to return to Kibana, and the error “problem with the configuration of authentication ” is shown.
+As you can see, Elasticsearch has no user-related information (metadata) to return to Kibana, and the error “problem with the configuration of authentication ” is shown.
 
-In general, we highly discourage implementing access control using origin IPs alone, users should set up SSL, Basic HTTP auth in their agents in any case, even on localhost. The hosts rule would then be an extra protection.
+In general, we highly discourage implementing access control using origin IPs alone, users should set up SSL, Basic HTTP auth in their agents in any case, even on localhost. The `hosts`` rule would then be an extra protection.
```
Comment (sscarduzio) [inline kibana.md:456]: double tick typo in `hosts`

File: kibana.md [hunk 4]
```diff
@@ -669,13 +669,13 @@ Enabling proxy auth passthrough will relax the requirement to provide a password
 
 This feature will work in all ReadonlyREST editions.
 
-Alternatively to typing in credentials in the standard login form, it is possible to create an authenticated Kibana session by passing a JWT token as a query parameter in a URL.
+Alternatively, to typing in credentials in the standard login form, it is possible to create an authenticated Kibana session by passing a JWT token as a query parameter in a URL.
```
Comment (sscarduzio) [inline kibana.md:672]: I don't understand this comma after alternatively

**Review by sscarduzio — COMMENTED** [review]
Massive improvement in typo busting and consistency! Did you use ChatGPT or did all by hand?

File: kibana.md (hunk 4 above)
Comment (coutoPL) [reply kibana.md]: you are right. I was too quick :P

File: kibana.md (hunk 2 above)
Comment (coutoPL) [reply kibana.md]: yeah. I've edited the whole sentence. WDYT?

**Comment by coutoPL** [comment]
@sscarduzio kindly reminder

**Review by sscarduzio — APPROVED** [review]

**Pull request merged**